account_cutoff_base/security/account_cutoff_base_security.xml

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data noupdate="0">
-        <record id="account_cutoff_multi_company_rule" model="ir.rule">
-            <field name="name">Account Cutoff Multi Company</field>
-            <field name="model_id" ref="model_account_cutoff"/>
-            <field name="global" eval="True"/>
-            <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
-        </record>
-    </data>
-</openerp>
+<odoo noupdate="1">
```

Comment:
This change is also in the version 9.0.
For me, record rules can be set as no updatable like it's done in Odoo core.
@alexis-via Any comment about this change ?